lib/streamlit/elements/arrow_altair.py

```diff
@@ -63,12 +63,21 @@
 
 
 class ChartType(Enum):
-    AREA = "area"
-    BAR = "bar"
-    LINE = "line"
-
-
-COLOR_LEGEND_SETTINGS = dict(titlePadding=0, offset=10, orient="bottom")
+    AREA = {"mark_type": "area"}
+    BAR = {"mark_type": "bar"}
+    LINE = {"mark_type": "line"}
+    SCATTER = {"mark_type": "circle"}
+
+
+# Color and size legends need different title paddings in order for them
+# to be vertically aligned.
+# NOTE: I don't think it's possible to *perfectly* align the size and
+# color legends in all instances, since the "size" circles vary in size based
+# on the data, and their container is top-aligned with the color container. But
+# through trial-and-error I found this value to be a good enough middle ground.
+# See e2e/scripts/st_arrow_scatter_chart.py for some alignment tests.
+COLOR_LEGEND_SETTINGS = dict(titlePadding=5, offset=5, orient="bottom")
```

Comment:
I think I missed this in the last PR but I think these settings should likely be defined here:
https://github.com/streamlit/streamlit/blob/develop/frontend/lib/src/components/elements/ArrowVegaLiteChart/CustomTheme.tsx

Reply:
These were already in Python when I first touched them, so I assumed there was a reason. But now I moved to CustomTheme.tsx and I can't tell the difference (even after comparing the screenshots in an image editor). So I'll leave it in CustomTheme.tsx.